src/client/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": "../../tsconfig.settings.json",
+    "compilerOptions": {
+        "target": "es5",
+        "lib": ["dom", "dom.iterable", "esnext"],
+        "strict": false,
+        "noEmit": true,
+        "allowJs": true,
+        "module": "esnext",
+        "isolatedModules": true,
+        "jsx": "preserve",
+        "baseUrl": ".",
```

Comment:
the baseUrl seems important here, won't resolve `'client/'` paths otherwise

Reply:
Note that I didn't actually remove the `baseUrl` in my PR - I lifted into a settings file at a higher level.  You absolutely do need a `baseUrl` if you're going to use path mappings.  My recommendation is basically to keep them together so it's clear where the path mappings resolve to.

Reply:
What I meant here is that lifting it to a higher level caused my client project not to resolve `client/*` imports. Adding it directly in this file fixed it, but I might've done other stuff wrong. Gonna give this another go soon

Reply:
And the reason not to lift the path mapping up as well is that only `client` imports from `client/*`?

Reply:
correct, only client imports from client/*